aoliver44/taxaHFE
# merge Development into main (#50)

* test actions (#46)

- adds github workflow that runs a test
- adds placeholder test with simple true == true
- adds badge to readme for workflow status

* Change abundance trim to not be outlier resistant

UPDATE: intially we had at 0.02, for an outlier resistant mean but if you want a = 0, p = 0 (all features), the trim was causing some features to get filtered (they look like zero abundance, but really the top 2% of features had non-zero abundances)

* version bumps

version bumps to keep track of newest version. This bump is for editing the "trim" parameter for the outlier resistant mean

* Leakage free taxa hfe (#48)

Merge taxaHFE-ML into the main development branch. It seems pretty stable in my testing. Documentation has been updated too in the readme.

* Update readme

abundance filter no longer does a "trimmed" mean

* Add a print statement for seed

Let the user see the seed

* Better writing of output files

For taxaHFE-ML:
- dont write standard output files
- wait to write test/train files for dietML script

* make taxaHFE-ML test faster with new switch

dont need to do lowest level on the test data generation (dont really need to do any competitions, but need to still fill out the trees...this is the fastest i think it can happen with the current code).

---------
